.gitignore

```diff
@@ -1,2 +1,3 @@
 .cargo
 /target
+target/
```

Comment:
Should this be in a separate .gitignore in examples/?

Reply:
Made a separate `.gitignore` in examples.